.github/workflows/amplify_canaries.yaml

```diff
@@ -156,3 +157,35 @@ jobs:
         run: |
           flutter build ios --simulator --target=integration_test/main_test.dart
           flutter test -d test integration_test/main_test.dart
+
+  log-failure-metric:
+    # Send failure data point to Cloudwatch metric CanaryTestFailure on failure
+    # See repo amplify-ci-support/github-cloudwatch-cdk for related CDK Cloudwatch setup
+    runs-on: macos-latest
```

Comment:
Maybe we can run on ubuntu-latest to save on costs.

Reply:
Yeah, I don't see any reason we'd run this on macos